broker.go

```diff
@@ -242,3 +242,26 @@ func sendMessage(s *discordgo.Session, channelID string, msg string) *discordgo.
 	}
 	return message
 }
+
+func sendMessageEmbed(s *discordgo.Session, channelID string, msg *discordgo.MessageEmbed) *discordgo.Message {
+	message, err := s.ChannelMessageSendEmbed(channelID, msg)
+	if err != nil {
+		log.Printf("failed to send message %+v to discord: %v", msg, err)
+	}
+	return message
+}
+
+func createMessageEmbed(ticker string, value float32) *discordgo.MessageEmbed {
+	prefix := ""
+	if test {
+		prefix = fmt.Sprintf("TEST(%s): ", messagePrefix)
```

Comment:
Move this logic to a helper function to be used in `sendMessage()` and `createMessageEmbed()`.

Reply:
ie `getPrefix()`